### PR TITLE
Add tagging query constant to rust bindings

### PR DESCRIPTION
**Pull request by silvergasp**

### CODE REVIEW

File: bindings/rust/lib.rs [hunk 1]
```diff
@@ -49,6 +49,9 @@ pub const GRAMMAR: &str = include_str!("../../grammar.js");
 /// The syntax highlighting query for this language.
 pub const HIGHLIGHT_QUERY: &str = include_str!("../../queries/highlights.scm");
 
+/// The symbol tagging query for this language.
+pub const TAGGING_QUERY: &str = include_str!("../../queries/tags.scm");
```
Comment (amaanq) [inline bindings/rust/lib.rs:53]:
```suggestion
pub const TAGS_QUERY: &str = include_str!("../../queries/tags.scm");
```

Comment (silvergasp) [reply bindings/rust/lib.rs:53]: Interestingly these names are somewhat inconsistent between crates e.g.
- https://docs.rs/tree-sitter-swift/latest/tree_sitter_swift/constant.TAGS_QUERY.html
- https://docs.rs/tree-sitter-rust/latest/tree_sitter_rust/constant.TAGGING_QUERY.html
- https://docs.rs/tree-sitter-javascript/latest/tree_sitter_javascript/constant.TAGGING_QUERY.html
- ...

Happy to change this, but it'd be nice to have a somewhat consistent naming convention for these. If we where to decide on one, do you think it'd be worth changing them in all the tree-sitter owned crates?

Comment (amaanq) [reply bindings/rust/lib.rs:53]: Let's stick with TAGS, I wasn't aware that rust and js used TAGGING, that likely predates when the default template switched to TAGS

**Comment by Squadrick** [comment]
@silvergasp Hey, checking if you had the time to change the variable name. This is needed for https://github.com/TabbyML/tabby/pull/744. 

If you're busy, let me know. I closed my PR #175 as a duplicate, and I can re-open it and merge that instead.

**Comment by silvergasp** [comment]
@Squadrick, this one just slipped under my radar, I've been in the process of wiping my dev environment and starting clean, so I've been a little preoccupied with other things. If you are blocked on this, and you've got a PR that's ready to go, I'd be happy to just defer to that instead. Also Tabby looks like a really cool project, and something that I will be looking into further :)

**Pull request closed without merging**